cloud/aws/alb/variables.tf

```
@@ -162,22 +168,16 @@ variable "latency_notifications_critical" {
   default     = []
 }
 
-variable "latency_aggregation_function" {
-  description = "Aggregation function and group by for latency detector (i.e. \".mean(by=['host'])\")"
-  type        = string
-  default     = ""
-}
-
-variable "latency_transformation_function" {
-  description = "Transformation function for latency detector (mean, min, max)"
-  type        = string
-  default     = "min"
+variable "latency_timer" {
```

Since unit is forced, variable name should indicate the unit.

```suggestion
variable "latency_timer_in_seconds" {
```

---

too long and already in description